docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -365,6 +365,8 @@ Standard ONNX\* operators:
 | ROIAlign | No |
 | Range | No |
 | Reciprocal | No |
+| ReduceL1 | No |
+| ReduceL2 | No |
 | ReduceMax | No |
 | ReduceMean | No |
```

Comment:
Add `EuclideanNorm` to the list of supported TF operations.

Reply:
Done.